samples/keypublisher/clients/cpix_client.py

```diff
@@ -0,0 +1,43 @@
+"""File containing abstract class for CPIX clients."""
+
+import abc
+import os
+
+from google.cloud import secretmanager
+
+
+class CpixClient(abc.ABC):
+  """Abstract class for CPIX clients."""
+
+  @abc.abstractmethod
+  def fetch_keys(self, media_id, key_ids):
```

Comment:
Please use type annotations in the method header. You already have the typing info in the docstring, but it's better to have it as annotations :)

Reply:
I've added the type hint here for media_id and key_ids., as well as some other function headers.

I saw that for noxfile_config.py, it's recommended to enforce type hint. There're a decent amount of methods with `self` as an argument, as well as more complex types. So I have for now changed `enforce_type_hints` to false and did not type annotate all methods. Please let me know your thoughts:)-- thanks!